src/include/update/rootpart.rb

```diff
@@ -63,12 +63,7 @@ def CanBeLinuxRootFS(partition_fs)
         return false
       end
 
-      # possible_root_fs contains list of supported FSs
-      # storage-ng
-      # FIXME: this kind of checks should be responsibility of the upcoming
-      # Y2Storage::FilesystemType (to be added as part of the
-      # "yast-storage-ng as a libstorage wrapper" change)
-      possible_root_fs = [:ext2, :ext3, :ext4, :btrfs, :reiser, :xfs]
+      possible_root_fs = Y2Storage::Filesystems::Type.root_filesystems().map { |x| x.to_sym }
       possible_root_fs.include?(partition_fs)
```

Comment:
In fact, we do need to expose the list for the widgets. But I fully agree we should have a method for this check. Not even a class method, but an instance method as suggested at https://github.com/yast/yast-update/pull/74#discussion_r121156552

If we plan to defer the creation of such method, the question is - how do we make sure we don't forget and that nobody finding this piece of temporary code adopts this as the correct way?